plugins/lsp/default.nix

```diff
@@ -134,6 +134,35 @@ in {
           };
           default = {};
         };
+
+        extra = mkOption {
+          type = with types; listOf helpers.keymaps.mapOptionSubmodule;
+          description = "Extra keymaps to register on 'LspAttach'.";
+          example = [
+            {
+              key = "<leader>lx";
+              action = "<CMD>LspStop<Enter>";
+            }
+            {
+              key = "<leader>ls";
+              action = "<CMD>LspStart<Enter>";
+            }
+            {
+              key = "<leader>lr";
+              action = "<CMD>LspRestart<Enter>";
+            }
+            {
+              key = "gd";
+              action = "require('telescope.builtin').lsp_definitions()";
+              lua = true;
+            }
+            {
+              key = "K";
+              action = "<CMD>Lspsaga hover_doc<Enter>";
```

Comment:
I wonder if this would add more confusion, since most users will use the "normal" hover?

Reply:
Yes, I just added that, and the telescope mapping as examples of why you would use this as well as how.

Easily removed if you think that'd be clearer.

Reply:
No strong opinions either way. More examples is usually good, this one just felt like it could potentially confuse new users.

I'll let the maintainers decide if it should be changed.

Reply:
I think this is fine now, with the updated description.